cdk/src/functions/enable-securityhub.ts

```diff
@@ -1,546 +1,114 @@
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-
-  return { disableStandardControlArns: disableStandardControls, enableStandardControlArns: enableStandardControls };
-}
-
-/**
- * Function to be executed before event specific action starts, this function makes the list of standards to enable or disable based on the input
- * @param securityHubClient
- * @param standardsToEnable
- * @param awsSecurityHubStandards
- */
-async function getStandardsModificationList(
-  securityHubClient: SecurityHub,
-  standardsToEnable: { name: string; enable: boolean; controlsToDisable: string[] | undefined }[],
-  awsSecurityHubStandards: { [name: string]: string }[],
-) {
-  const existingEnabledStandards = await getExistingEnabledStandards(securityHubClient);
-  const toEnableStandardRequests = [];
-  const toDisableStandardArns: string[] | undefined = [];
+export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent) {
+  const homeRegion = process.env.homeRegion!;
+  const adminAccountId = process.env.adminAccountId!;
+  const secHubCrossAccountRoleArn = process.env.role!;
 
-  // if no standard provided to enable, then disable all existing enabled standards
-  if (!standardsToEnable || standardsToEnable.length === 0) {
-    for (const existingEnabledStandard of existingEnabledStandards) {
-      toDisableStandardArns.push(existingEnabledStandard?.StandardsSubscriptionArn);
-    }
+  if (!adminAccountId || !secHubCrossAccountRoleArn || !homeRegion) {
+    throw new Error('homeRegion, adminAccountId and role env variable is required');
   }
 
-  // for each standard to enable, check if it is already enabled, if not then add it to enable list, else to disable list
-  for (const inputStandard of standardsToEnable) {
-    if (inputStandard.enable) {
-      for (const awsSecurityHubStandard of awsSecurityHubStandards) {
-        if (awsSecurityHubStandard[inputStandard.name]) {
-          const existingEnabledStandard = existingEnabledStandards.filter(
-            (item) => item.StandardsArn === awsSecurityHubStandard[inputStandard.name],
-          );
-          if (existingEnabledStandard.length === 0) {
-            toEnableStandardRequests.push({ StandardsArn: awsSecurityHubStandard[inputStandard.name] });
-          }
-        }
-      }
-    } else {
-      for (const awsSecurityHubStandard of awsSecurityHubStandards) {
-        if (awsSecurityHubStandard[inputStandard.name]) {
-          const existingEnabledStandard = existingEnabledStandards.find(
-            (item) => item.StandardsArn === awsSecurityHubStandard[inputStandard.name],
-          );
-
-          if (existingEnabledStandard) {
-            toDisableStandardArns.push(existingEnabledStandard?.StandardsSubscriptionArn);
-          }
-        }
-      }
-    }
+  let eventType = event.RequestType;
+  if (!eventType) {
+    // setting RequestType to Create or Update since lambda is not invoked by CloudFormation
+    eventType = 'Update';
```

Comment:
I think it would be cleaner to check for the expected cloudtrail event here. Later in the condition you can also check 
`if event == Cloudtrail.UpdateLandingZoneEvent || event.RequestType == 'Create' || event.RequestType == 'Update'` or simila